taxonomies/labels.txt

```diff
@@ -474,6 +474,7 @@ ru:EAC
 wikidata:en:Q18407253
 
 fr:Entrepreneurs + Engagés, E+, Entrepreneurs Engagés
+de:Entrepreneurs + Engagés
```

Comment:
Why add this?

Reply:
Some values are stored in German, for some reason 🤷

Reply:
I think that's the usual disruptive apps, probably some older changes, but we shouldn't update the taxonomy to reflect that IMHO. Not having a taxonomy entry for this actually makes it easier to fix the values.

Reply:
you are perfectly right, I closed the PR